### ISSUE TITLE
Unknown layer type: Hardtanh, known types: ['Slice', 'MaxPool2d', 'Add', 'PReLU', 'ELU', 'MulConstant', 'Dropout', 'Addmm', 'BatchNorm', 'Concat', 'UpsamplingBilinear2d', 'Tanh', 'Softmax', 'ConvNd', 'Cmax', 'data', 'View', 'AddConstant', 'Sigmoid', 'LeakyReLU', 'AvgPool2d', 'Threshold']

### ISSUE THREAD

**Issue by abc8350712** [starimeL/PytorchConverter#20]
The version of pytorch is '0.2.0+c1b09cd'.However, the problem is still coming.
And my model is mobilenetv2.
Could you help me?

**Comment by zcl912** [comment]
> The version of pytorch is '0.2.0+c1b09cd'.However, the problem is still coming.
> And my model is mobilenetv2.
> Could you help me?

hello, have you ever solve the problem, i met the same error
